content/getting-started/adding-apps-from-custom-sources.md

```diff
@@ -4,33 +4,48 @@ title: Adding apps from custom sources
 weight: 4
 ---
 
-You can add apps from public or private Git-based repositories. This includes repositories requiring **SSH key authentication**. 
+You can add apps from public or private Git-based repositories. 
 
 {{<notebox>}}
 Note that apps added from custom sources have some limitations.
 
 * Webhooks for automatic build triggering need to be [set up manually](../building/automatic-build-triggering/#webhooks).
-* [Automatic builds](../building/automatic-build-triggering) are available on code push and tag creation but not on pull request creation or update.   
+* Using `codemagic.yaml` for build configuration is currently not supported.  
 {{</notebox>}}
 
-To add an app from custom source:
+## Connecting repository via HTTP/HTTPS
 
 1. Click **Add app from custom source** on the Applications page.
 
-    {{< figure size="" src="../uploads/add-app-from-custom-source1.png" caption="" >}}
+    {{< figure size="medium" src="../uploads/add-app-from-custom-source1.png" caption="" >}}
 
-2. Then, fill in all the required fields.
+2. Enter the **Repository URL** for **cloning** the repository. You can find the URL from your repository on clicking **Clone** / **Clone or download**. Usually, the URL is in this format: `https://example.com/username/repo.git`.
```

Comment:
Consider using `https://example.com/path/to/repository.git`. Using `username` may look like something specific.